app/src/main/java/com/github/wnder/Storage.java

```diff
@@ -25,35 +25,31 @@
 import java.util.Set;
 import java.util.concurrent.CompletableFuture;
 
-public class Storage{
-    private FirebaseFirestore db;
-    private FirebaseStorage storage;
-    private StorageReference storageRef;
+public final class Storage{
 
-
-    public Storage(){
```

Comment:
You might want to add a private constructor to avoid anyone making a instance of this class by mistake.